### ISSUE TITLE
Error: Schema must contain unique named types but contains multiple types named "ID".

### ISSUE THREAD

**Issue by artem-barmin** [calebmer/postgraphql#33]
Tried to extend generated schema, got error from header. 

Code to reproduce(working in modified main.js from this project):
`import {parse, extendSchema} from 'graphql';`
...
Before running server: `const extendedSchema = extendSchema( graphqlSchema,parse('extend type RootQuery{ id1: String}'));`

During debugging found that IDType from 'types.js' clashes with default 'ID' defined internally. For some reason this validation only failing when I am extending created schema.

Following debug output is from [https://github.com/graphql/graphql-js/blob/master/src/type/schema.js#L222](https://github.com/graphql/graphql-js/blob/master/src/type/schema.js#L222) after calling `extendSchema` function :

```map[type.name] = GraphQLScalarType {
  name: 'ID',
  description: 'A globally unique identifier used to refetch an object or as a key for a cache. It is not intended to be human readable.',
  _scalarConfig: 
   { name: 'ID',
     description: 'A globally unique identifier used to refetch an object or as a key for a cache. It is not intended to be human readable.',
     serialize: [Function],
     parseValue: [Function],
     parseLiteral: [Function] } } ```

```type = GraphQLScalarType {
  name: 'ID',
  description: 'The `ID` scalar type represents a unique identifier, often used to refetch an object or as key for a cache. The ID type appears in a JSON response as a String; however, it is not intended to be human-readable. When expected as an input type, any string (such as `"4"`) or integer (such as `4`) input value will be accepted as an ID.',
  _scalarConfig: 
   { name: 'ID',
     description: 'The `ID` scalar type represents a unique identifier, often used to refetch an object or as key for a cache. The ID type appears in a JSON response as a String; however, it is not intended to be human-readable. When expected as an input type, any string (such as `"4"`) or integer (such as `4`) input value will be accepted as an ID.',
     serialize: [Function: String],
     parseValue: [Function: String],
     parseLiteral: [Function: parseLiteral] } }```

This can be easily solved by renaming `ID` type to `EntityID`

**Comment by artem-barmin** [comment]
Just realized that `extendSchema` produces client schema, that can't be used for execution. But anyway, this error can arrive in other scenarios

**Comment by calebmer** [comment]
The name `ID` is actually required by Relay 😖

I think the best solution is to just use the GraphQL `ID` type. I'll make this change.

**Comment by calebmer** [comment]
Fixed in `1.2.3` 👍
